### PR TITLE
Fix typo in uninstall.php

### PR DESCRIPTION
**Pull request by aaemnnosttv**
## Summary

Addresses issue #2311 

## Relevant technical choices

- Confirmed this fixes the error
- Updated QA brief on issue

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: uninstall.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 }
 
 // Load plugin main file to bootstrap infrastructure and add hooks.
-require_once dirname( __FILE ) . '/google-site-kit.php';
+require_once dirname( __FILE__ ) . '/google-site-kit.php';
```
Comment (eugene-manuilov) [inline uninstall.php:17]: @aaemnnosttv maybe we should use just `__DIR__`?
```suggestion
require_once __DIR__ . '/google-site-kit.php';
```

Comment (aaemnnosttv) [reply uninstall.php:17]: Yeah, the result there is the same and a little cleaner so generally I would opt for that as well, but given that this needs to have maximum compatibility with older PHP versions (just like `google-site-kit.php`) I think it's marginally better to keep the older style here. Given that I think `__DIR__` was introduced in PHP 5.3, it's not really a big concern, but I'm guessing this is probably what @felixarntz was thinking here.

Comment (eugene-manuilov) [reply uninstall.php:17]: Ok, not a big deal though. We can use `dirname`.

**Review by eugene-manuilov — APPROVED** [review]

**Review by felixarntz — APPROVED** [review]
OMG 🤦‍♂️  Thanks for fixing this!

**Pull request merged**